torch/ao/quantization/quantization_mappings.py

```diff
@@ -112,12 +112,13 @@
     nni.LinearReLU: nniqd.LinearReLU,
     nn.EmbeddingBag: nnq.EmbeddingBag,
     nn.Embedding: nnq.Embedding,
-    nn.Conv1d: nnqd.Conv1d,
-    nn.Conv2d: nnqd.Conv2d,
-    nn.Conv3d: nnqd.Conv3d,
-    nn.ConvTranspose1d: nnqd.ConvTranspose1d,
-    nn.ConvTranspose2d: nnqd.ConvTranspose2d,
-    nn.ConvTranspose3d: nnqd.ConvTranspose3d,
+    # Don't want to enable these by default because the performance is poor compared to other dynamic ops
```

Comment:
nit: do we mean "accuracy"?  performance is usually used for "inference time" in this context

Reply:
yeah, i'll fix that